test/servico/cst02Test.java

```diff
@@ -64,7 +61,7 @@ public void tearDown() {
         List<AusenciaModel> ausencias = notificaçãoService.listarAusencias();        
         assertEquals(0, ausencias.size());
         
-        notificaçãoService.notificarAusencia(professor.id, "25/11/2013", "25/11/2013", "Motivo Declarado", new LinkedList<String>());
+        notificaçãoService.notificarAusencia(professor.id, "25/11/2013", "25/11/2013", "Palestra", new LinkedList<String>());

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 125).](https://www.codacy.com/app/calebepb/substituicao/file/2766648423/issues/source?bid=3308597&fileBranchId=3353492#l64)